### PR TITLE
Let users choose their multiregion for GCS benchmarking

### PR DESCRIPTION
**Pull request by noahl**
As the title says.

### CODE REVIEW

**Comment by noahl** [comment]
I've tested this in the US and Asia, and it worked fine. I'm running a Europe test right now, but I don't expect any surprises.

@yuyantingzero @stfeng please take a look.

**Comment by noahl** [comment]
Update: the Europe test succeeded. That covers all GCS multiregions.

File: perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py [hunk 1]
```diff
@@ -865,7 +868,11 @@ def Prepare(self, vm):
 
     vm.bucket_name = 'pkb%s' % FLAGS.run_uri
 
-    vm.RemoteCommand('%s mb gs://%s' % (vm.gsutil_path, vm.bucket_name))
+    make_bucket_command = '%s mb' % vm.gsutil_path
+    if FLAGS.object_storage_gcs_multiregion:
```
Comment (stfeng) [inline perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py]: Let's output the gcs multi-region location name, and the regional location name (of all 3 providers) as a metadata, for example - 

metadata['gcs_multiregion_location'] = "default", or whatever the user has specified via the flag.

metadata['regional_bucket_location'] = "default", or whatever the user has specified via FLAGS.object_storage_region (for all 3 providers)

This way, when pkb users collect and analyze data from each run, they can categorize based on bucket locations and compare properly.

**Comment by stfeng** [comment]
Just one comment about adding metadata entries for bucket location.

Thanks for the extra effort in testing.

File: perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py (hunk 1 above)
Comment (noahl) [inline perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py]: Done. I tested it on GCP, AWS, and Azure.

**Comment by noahl** [comment]
@stfeng @yuyantingzero I added the bucket locations to the metadata. Please take a look.

**Comment by stfeng** [comment]
LGTM

**Pull request merged**